### ISSUE TITLE
Mark maven plugin as thread safe

### ISSUE THREAD

**Issue by Dunemaster** [spring-io/spring-javaformat#133]
Currently, the plugin is not marked thread-safe which results in the following message when building, for example, spring boot using parallel builder

```
$ mvnw clean package -T3 -DskipTests --offline

[WARNING] *****************************************************************
[WARNING] The following plugins are not marked @threadSafe in Spring Boot Docs:
[WARNING] *****************************************************************
[WARNING] * Your build is requesting parallel execution, but project      *
[WARNING] * contains the following plugin(s) that have goals not marked   *
[WARNING] * as @threadSafe to support parallel building.                  *
[WARNING] * While this /may/ work fine, please look for plugin updates    *
[WARNING] * and/or request plugins be made thread-safe.                   *
[WARNING] io.spring.javaformat:spring-javaformat-maven-plugin:0.0.15
[WARNING] Enable debug to see more precisely which goals are not marked @threadSafe.
[WARNING] *****************************************************************
[WARNING] * If reporting an issue, report it against the plugin in        *
[WARNING] * question, not against maven-core                              *
[WARNING] *****************************************************************
[WARNING] The following plugins are not marked @threadSafe in Spring Boot Actuator Starter:
[WARNING] io.spring.javaformat:spring-javaformat-maven-plugin:0.0.15
[WARNING] Enable debug to see more precisely which goals are not marked @threadSafe.
[WARNING] *****************************************************************
```

**Comment by Dunemaster** [comment]
I looked through the code, and it seems thread-safe for me, so it should be just a matter of setting threadsafe = true in the @Mojo annotations

**Comment by philwebb** [comment]
Fixed in f57fa9247f671d9c58ba8b1d1a4ec4473d905d5e (accidentally referenced the wrong issue).
